src/plugins/intel_cpu/src/emitters/snippets/aarch64/jit_kernel_emitter.hpp

```diff
@@ -0,0 +1,164 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "emitters/plugin/aarch64/jit_emitter.hpp"
+
+#include "jit_container_emitter.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace aarch64 {
+
+#define SNIPPETS_MAX_SNIPPETS_DIMS 12
```

Comment:
Applied. BTW I revised the member variables like m_work_amount to be public, otherwise there will be compilation errors "m_work_amount is private within this context". The reason seems to be GET_OFF_LOOP_ARGS can only access to private member variable when the class definition and GET_OFF_LOOP_ARGS calling are in the same namespace. I.E. if the class is also define inside ov::intel_cpu::**aarch64**, then there won't be errors. I simply use public, so we can still use GET_OFF_LOOP_ARGS. Another alternative approach I think is add member function get_work_amount(). Please feel free to have further discussions, if you have concerns. Thanks Alexandra!